src/Simple.OData.Client.Core/Adapter/IBatchWriter.cs

```diff
@@ -10,7 +10,7 @@ namespace Simple.OData.Client
     public interface IBatchWriter
     {
         Task<ODataRequest> CreateBatchRequestAsync(IODataClient client, IList<Func<IODataClient, Task>> actions, IList<int> responseIndexes);
-        Task<object> CreateOperationMessageAsync(Uri uri, string method, string collection, IDictionary<string, object> entryData, bool resultRequired);
+        Task<object> CreateOperationMessageAsync(Uri uri, string method, string collection, IDictionary<string, object> entryData, IDictionary<string, string> operationHeaders, bool resultRequired);
```

Comment:
This is a breaking change to a public interface.

Reply:
Same as above